starter-project-web-vue2/components/bisrat/AddEditBlog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<div style="margin:auto;max-width:600px !important;padding-bottom:30px">
+  <h2><span v-if="id==null">Add Blog</span><span v-if="id!=null">Update Blog</span></h2>
+  
+  <form>
+    <v-text-field
+      v-model="title"
+	  label="Title"
+    ></v-text-field>
+    <v-textarea
+      v-model="content"
```

Comment:
nit; correct indentation makes the code more readable. you can use `npm run lintfix`